lib/scrabble_word.rb

```diff
@@ -0,0 +1,34 @@
+require "pry"
+
+class ScrabbleWord
+
+  def initialize (word)
+    @word = word
```

Comment:
This works! You could also `downcase` the word upon initialization and use an `attr_reader` for a little cleaner syntax